index.html

```diff
@@ -17,6 +17,10 @@
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
         integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
 
+    <!-- custome css for home page -->
+    <link rel="stylesheet" href="css/home_style.css">
```

Comment:
@codervivek5 you have to use `styles.css` to style the elements in `index.html` . Don't need to create a new css file . Put the css in `styles. css` file only

Reply:
Thanks @unnikrishnan2002 for being so helpful in supporting your fellow mentee. Appreciate your willingness to do so.

Are you able to do so @codervivek5

Reply:
Yes i will do that...